src/components/common/Extensions/UserHub/partials/TransactionsTab/TransactionsTab.tsx

```diff
@@ -8,8 +8,6 @@ import { useTransactionsListObserver } from './hooks';
 import TransactionList from './partials/TransactionList';
 import { TransactionsProps } from './types';
 
-import styles from './TransactionsTab.css';
```

Comment:
if we are not going to use the styles provided by the stylesheet, then we should remove it as well seeing that this is the only component that uses it.

Reply:
Correct. Remove that import, and if you so happen to need the styles declared in there, they should be transitioned over to tailwind